ios/en.lproj/Localizable.strings

```diff
@@ -5,4 +5,4 @@
   Created by Minh DrMinh on 7/19/23.
   
 */
-"SINGLE_UNREAD" = "1 unread chat(s)";
+"SINGLE_UNREAD" = "You have 1 encrypted message";
```

Comment:
you also update or add SINGLE_UNREAD in `vi.lproj/Localizable.strings` and `fr.lproj/Localizable.strings`